tools/check_headers/stumpless.yml

```diff
@@ -452,3 +452,4 @@
 "winsock2_network_target_is_open": "private/config/have_winsock2.h"
 "stumpless_get_cons_stream" : "stumpless/target.h"
 "stumpless_set_cons_stream" : "stumpless/target.h"
+"stumpless_log_to_error_stream" : "stumpless/error.h"
```

Comment:
move the function entry (with the new name and include file) into the `stumpless_private.yml` file instead